src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -0,0 +1,3394 @@
+          energyCost P, C
+          attackRequirement {}
+          onAttack {
+            damage 60
+          }
+        }
+      };
+      case MIMIKYU_82:
+      return basic (this, hp:HP070, type:P, retreatCost:1) {
+        weakness D
+        resistance F, MINUS30
+        bwAbility "Heal Stop", {
+          text "Your opponent’s Benched Pokémon can’t be healed."
+          actionA {
+          }
+        }
+        move "Claw Slash", {
+          text "30 damage. "
+          energyCost P, C
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+      };
+      case SINISTEA_83:
+      return basic (this, hp:HP030, type:P, retreatCost:1) {
+        weakness D
+        resistance F, MINUS30
+        move "Furtive Drop", {
+          text "Put 2 damage counters on your opponent’s Active Pokémon."
+          energyCost C
+          attackRequirement {}
+          onAttack {
+
+          }
+        }
+      };
+      case POLTEAGEIST_84:
+      return evolution (this, from:"Sinistea", hp:HP060, type:P, retreatCost:1) {
+        weakness D
+        resistance F, MINUS30
+        bwAbility "Tea Break", {
+          text "Once during your turn, you may discard a Pokémon with a Mad Party attack from your hand and draw 2 cards."
+          actionA {
+          }
+        }
+        move "Mad Party", {
+          text "20x damage. This attack does 20 damage for each Pokémon in your discard pile with a Mad Party attack."
+          energyCost C, C
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+      };
+      case DIGLETT_85:
+      return basic (this, hp:HP050, type:F, retreatCost:1) {
+        weakness G
+        move "Scratch", {
+          text "20 damage. "
+          energyCost F
+          attackRequirement {}
+          onAttack {
+            damage 20
+          }
+        }
+      };
+      case DUGTRIO_86:
+      return evolution (this, from:"Diglett", hp:HP090, type:F, retreatCost:1) {
+        weakness G
+        move "Dig", {
+          text "30 damage. Flip a coin. If heads, prevent all effects of attacks, including damage, done to this Pokémon during your opponent’s next turn."
+          energyCost F
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+        move "Mud Bomb", {
+          text "70 damage. "
+          energyCost F, C
+          attackRequirement {}
+          onAttack {
+            damage 70
+          }
+        }
+      };
+      case GALARIAN_FARFETCH_D_87:
+      return basic (this, hp:HP010, type:C, retreatCost:2) {
+        weakness Y
+        resistance F, MINUS30
+        bwAbility "TBA", {
+          text "TBA"
+          actionA {
+          }
+        }
+        move "TBA", {
+          text "100 damage. TBA"
+          energyCost C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 100
+          }
+        }
```

Comment:
Not reaaaally relevant but maybe add a comment about rewriting this whole card once revealed?

```suggestion
      case GALARIAN_FARFETCH_D_87: //TODO: Get data on this card.
      return basic (this, hp:HP010, type:C, retreatCost:2) {
        weakness Y
        resistance F, MINUS30
        bwAbility "TBA", {
          text "TBA"
          actionA {
          }
        }
        move "TBA", {
          text "100 damage. TBA"
          energyCost C, C, C
          attackRequirement {}
          onAttack {
            damage 100
          }
        }
```